resteasy-dependencies-bom/pom.xml

```diff
@@ -22,15 +22,15 @@
         <version.com.fasterxml.jackson>2.9.8</version.com.fasterxml.jackson>
         <version.com.google.inject.guice>4.1.0</version.com.google.inject.guice>
         <version.com.io7m.xom>1.2.10</version.com.io7m.xom>
-        <version.com.sun.mail>1.5.6</version.com.sun.mail>
+        <version.com.sun.mail>1.6.2</version.com.sun.mail>
         <version.org.glassfish.jaxb.jaxb>2.3.1</version.org.glassfish.jaxb.jaxb>
         <version.com.sun.xml.fastinfoset>1.2.13</version.com.sun.xml.fastinfoset>
         <version.commons-io.commons-io>2.5</version.commons-io.commons-io>
         <version.hamcrest>1.3</version.hamcrest>
         <version.io.netty.netty>3.10.6.Final</version.io.netty.netty>
         <version.io.netty.netty4>4.1.31.Final</version.io.netty.netty4>
-        <version.io.vertx>3.5.4</version.io.vertx>
-        <version.io.undertow>1.4.18.Final</version.io.undertow>
+        <version.io.vertx>3.6.2</version.io.vertx>
+        <version.io.undertow>1.4.26.Final</version.io.undertow>
```

Comment:
Why not move directly to undertow 2.x.x ? WildFly 12+ is using this stream.

Reply:
RESTEasy project has Undertow integration so it's not just about WF based version of Undertow - https://github.com/resteasy/Resteasy/tree/master/server-adapters/resteasy-undertow. I didn't want to do major bumps in this PR.

@asoldano Can we jump to 2.0.z series ?

Reply:
Sure but there is only one integration class in resteasy and undertow 2.0.z should be compatible with 1.4.z https://abi-laboratory.pro/index.php?view=archives_report&lang=java&l=undertow-core&v1=1.4.26&v2=2.0.16

Reply:
Moving to Undertow 2

Reply:
fine, thanks